eba.opam

```diff
@@ -1,4 +1,4 @@
-opam-version: "1.2"
```

Comment:
This change should probably be part of PR #3, in case you want to keep two separate PR. For me it's fine if you squash them.